src/api/handlers/clients.js

```diff
@@ -0,0 +1,15 @@
+import Axios from 'axios';
+
+const options = {
+  method: 'GET',
+  headers: {
+    'x-api-key' : 'YsO9MGt0DwUu2PDTQ4y4TrhRtnJlYFzYWa1NW92PmvDFqcyDGLiacSNDWape'
+  }
+};
+
+const getAllClients = async () => {
+  const response = await Axios.get('https://658m0jk5l3.execute-api.us-east-1.amazonaws.com/prod/clients/get', options);
```

Comment:
This url also should be moved, then instead we can say make request with /clients/get as the only part we need to append to the end

Reply:
This is also an environmental variable!

Here's the code for an example request using the environment variables that have been setup:

```
`const options = {
  method: 'GET',
  headers: {
    'x-api-key' : process.env.AUTHENTICATION_TOKEN
  }
};

const getAllProjects = async () => {
  const response = await Axios.get(`${process.env.API_URL}/projects/get`, options);
  return response;
};`

```